packages/mobile/src/store/sagas.ts

```diff
@@ -1,15 +1,13 @@
 import remoteConfig from 'audius-client/src/common/store/remote-config/sagas'
-import { all, fork } from 'typed-redux-saga/macro'
-
-import { remoteConfigInstance } from 'app/services/remote-config/remote-config-instance'
+import { all, fork } from 'typed-redux-saga'
```

Comment:
Why do we not need the `/macro` here?

Reply:
there is weird behavior with /macro when using across web and mobile, so for consistency rn, removing macro usages until we move everything to common